integ-test/build.gradle

```diff
@@ -97,15 +102,40 @@ compileTestJava {
 
 testClusters.all {
     testDistribution = 'archive'
+    keystore 'plugins.query.federation.catalog.config', new File("$projectDir/src/test/resources/catalog/", 'catalog.json')
 }
 
 testClusters.integTest {
     plugin ":opensearch-sql-plugin"
 }
+task startPrometheus(type: SpawnProcessTask) {
+    doFirst {
+        download.run {
+            src 'https://github.com/prometheus/prometheus/releases/download/v2.39.1/prometheus-2.39.1.linux-amd64.tar.gz'
```

Comment:
What is the best practice here? Should I rely on github links or S3 links.

Reply:
wondering how often this needs to be changed?

Reply:
Till the next major release. I will be sticking to this version.

Reply:
should this test be optional? I assume it won't work on arm64 and windows/mac

Reply:
I have seen derek's PR, it removed doctests and integTests. When we fix for them we can use mac/windows binaries for them.
or we can create a separate task including tests for prometheus.

Reply:
yes that's ok then, the integTest/docTests should be fixed when 2.4 is out, then this can be changed to support other platforms

Reply:
https://github.com/opensearch-project/sql/issues/982